backend/src/graphql/resolver/ContributionMessageResolver.ts

```diff
@@ -142,17 +152,23 @@ export class ContributionMessageResolver {
         lastName: contribution.user.lastName,
         email: contribution.user.emailContact.email,
         language: contribution.user.language,
-        senderFirstName: user.firstName,
-        senderLastName: user.lastName,
+        senderFirstName: moderator.firstName,
+        senderLastName: moderator.lastName,
         contributionMemo: contribution.memo,
       })
       await queryRunner.commitTransaction()
+      await EVENT_ADMIN_CONTRIBUTION_MESSAGE_CREATE(
+        { id: contribution.userId } as DbUser,
```

Comment:
the userId is overwritten by the moderatorId in line 136 and cause here a wrong event entry

Reply:
This is not correct - like 136 is about `contributionMessage` not `contribution.userId `. The `contribution.userId` is not touched.

`contributionMessage.userId = moderator.id`